python/lsst/ts/MTAOS/MtaosCsc.py

```diff
@@ -19,25 +19,22 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-import time
-import warnings
+__all__ = ["MtaosCsc"]
+
+import inspect
 import asyncio
 import concurrent
 
 from lsst.ts import salobj
-
-from lsst.ts.MTAOS.ConfigByObj import ConfigByObj
-from lsst.ts.MTAOS.Model import Model
-from lsst.ts.MTAOS.ModelSim import ModelSim
-from lsst.ts.MTAOS.Utility import getSchemaDir, getCscName, WEPWarning, \
-    OFCWarning
+from lsst.ts import MTAOS
```

Comment:
I am afraid I was not clear.  It is not safe to use lsst.ts.MTAOS because it can lead to circular imports. Both your old code and this replacement have this problem.

I **strongly** suggest you use the safer alternative: relative imports of the other MTAOS modules that you need. All modules in a Python library should work this way (at least that is a good "rule of thumb").

 The following is my suggested replacement. This will act like the old code, except symbols from Utility will need a Utility prefix, e.g. Utility.getSchemaDir. You could use `from .Utility import getSchemaDir, ...` if you prefer, but I think it is clearer to be explicit for the many symbols in Utility:
```
from . import ConfigByObj import ConfigByObj
from . import Model  # and use Model.foo to access symbol foo
from . import ModelSim
from . import Utility
```

Reply:
OK. After checking the `ts_salobj`, I realized what do you want me to do. You want me to use the relative import for the python classes. But use the absolute import for the test script. Is this correct?
Will do for this although I thought the absolute path (my original method based on PEP-8) is better. But I think it is also good to align with the DM packages. Thanks!

Reply:
Exactly. Note: we discussed this privately. I explained that using relative imports for code *within* a library works helps with circular import issues (however, Python may have fixed those issues, and if so, absolute imports are certainly nicer).

Reply:
I just tried a google search and came up empty. I've also asked on the Slack software-dev channel. If the current code works and is likely to keep working then it'd be a shame to change it.